wp-content/themes/core/blocks/tribe/logo-marquee/editor.pcss

```diff
@@ -0,0 +1,5 @@
+/**
+ * The following styles get applied inside the editor only.
+ *
+ * Replace them with your own styles or remove the file completely.
+ */
```

Comment:
❓ I noticed the editor view hides most of the images. Do we need to disable the overflow for the editor view so that the user can see all of the images added to the gallery, or do you think having the view be correct in the editor is more important than being able to actually use the "delete" function added in `edit.js` for every image?

IMO if we're going for more of a "here's a preview of what the block would be on the FE within the editor", I would remove the "delete" function from the images. Otherwise, we should fit the images in the editor into a grid instead of allowing them to overflow.